### ISSUE TITLE
The example Show_straight_skeleton result in Segmentation fault

### ISSUE THREAD

**Issue by account-related** [CGAL/cgal#4581]
## Issue Details

I'm trying out CGAL for the first time, and when I try some examples I get _Segmentation fault_. These are the things I've done:

- Installed the prerequisites according to https://doc.cgal.org/latest/Manual/usage.html#secprerequisites
- Downloaded CGAL 5.0.2 from the releases of this repo
- Unzipped, and went into the `Straight_skeleton_2` examples
- Compiled using exact instructions given at https://doc.cgal.org/latest/Manual/usage.html#usage_introduction

So, this is what I did:
```
cd $HOME/CGAL-5.0.2/examples/Straight_skeleton_2
cmake -DCMAKE_BUILD_TYPE=Release .
make
```
No compilation errors.

When I try the resulting `Show_straight_skeleton` binary with the four example dat files, two result in Segmentation fault:
```
./Show_straight_skeleton sample_1.dat sample_1.eps
Input file: sample_1.dat
Result: sample_1.eps

./Show_straight_skeleton sample_2.dat sample_2.eps
Input file: sample_2.dat
Segmentation fault

./Show_straight_skeleton sample_3.dat sample_3.eps
Input file: sample_3.dat
Result: sample_3.eps

./Show_straight_skeleton sample_4.dat sample_4.eps
Input file: sample_4.dat
Segmentation fault
```

Considering that two eps files are produced, and they seem perfectly reasonable, I think I probably haven't messed up somewhere. (But I'm not overly certain.) I read in some other issue here that CGAL sometimes outputs _Segmentation fault_ if it simply dislikes the input values. So I'm thinking that perhaps there's a bug somewhere, or perhaps sample_2.dat and sample_4.dat have a problem in them?

## Source Code

I have not made any changes to any source code, I'm just trying to run one of the provided examples.

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Linux (Debian 10.3)
* Compiler: g++ (8.3.0)
* Release or debug mode: Release
* Specific flags used (if any):
* CGAL version: 5.0.2
* Boost version: 1.67
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by account-related** [comment]
I just wanted to add that a reasonable first step might be for somebody in the CGAL team, _with a known good installation_, to run the examples above and see if they also get the Segmentation fault.

I mean, if it works for them, perhaps I just messed up something when I installed it all?

**Comment by sloriot** [comment]
sample2 was not using the correct input format (I fixed it in the repo) and sample4 had some orientation issue that is fixed in the example code in #4582
